### PR TITLE
Yast2 squid tests fix

### PR DESCRIPTION
**Pull request by AdamMajer**
Squid requires forwarding ports when configured as a transparent proxy. If only transparent proxy ports are present, the service will fail to start with an error message.

These changes fix the error and add better logging in case of any future test failures.

- Related ticket: https://bugzilla.novell.com/show_bug.cgi?id=1077366
- Needles: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/316
- Verification run: http://tortuga.suse.de/tests/161

### CODE REVIEW

File: tests/console/yast2_proxy.pm [hunk 1]
```diff
@@ -50,11 +50,19 @@ sub empty_field {
...
+    upload_logs("/var/log/squid/proxy_cache.log");
+    upload_logs("/var/log/squid/proxy_store.log");
+}
+
 sub run {
     select_console 'root-console';
     if (is_sle && sle_version_at_least('15')) {
-        my $ret = zypper_call('in squid', exitcode => [0, 104]);
-        return record_soft_failure 'bsc#1056793' if $ret == 104;
+        my $ret = zypper_call('in squid');
```
Comment (rwx788) [inline tests/console/yast2_proxy.pm]: Let's remove variable, as not used anymore

File: tests/console/yast2_proxy.pm [hunk 2]
```diff
@@ -104,6 +112,17 @@ sub run {
     # move to http ports
     select_sub_menu 'start_up', 'http_ports';
 
+    # add a forwarding port
```
Comment (rwx788) [inline tests/console/yast2_proxy.pm]: Does it also work on SLE 12 SP4? And I guess we need MR for SLE needles.

File: tests/console/yast2_proxy.pm (hunk 1 above)
Comment (AdamMajer) [reply tests/console/yast2_proxy.pm]: Fixed now.

File: tests/console/yast2_proxy.pm (hunk 2 above)
Comment (AdamMajer) [reply tests/console/yast2_proxy.pm]: I'll get back to you on this one. It *should*

Comment (AdamMajer) [reply tests/console/yast2_proxy.pm]: Yes, works just fine with SLE 12 SP4 tests.

http://tortuga.suse.de/tests/169

Comment (rwx788) [reply tests/console/yast2_proxy.pm]: Great, I'll merge then. Thanks!

**Comment by okurz** [comment]
I think there was never a submission to SLE needles repo, right? That can explain https://progress.opensuse.org/issues/32428 . I will create the according needle(s) based on the tortuga reference.

**Pull request merged**